app/services/decisions/back_decision_tree.rb

```diff
@@ -29,6 +29,8 @@ class BackDecisionTree < DslDecisionTree
     from('nsm/steps/reason_for_claim')
       .when(-> { application.can_claim_youth_court? })
```

Comment:
@patrick-laa the above change is literally just for here so we can determine if we should go back to the outcome screen or the "Do you want to claim this?" screen